.eslintrc

```diff
@@ -1,34 +1,6 @@
 {
-  "extends": "airbnb",
-  "parser": "babel-eslint",
-  "rules": {
-    "arrow-parens": [2, "as-needed"],
-    "import/no-extraneous-dependencies": ["error", {"devDependencies": ["**/*.test.js", "**/*.spec.js"]}],
-    "indent": ["error", "tab", { "SwitchCase": 1 }],
-    "new-cap": [
-        2,
-        {
-            "capIsNewExceptions": ["List", "Map"]
-        }
-    ],
-    "no-mixed-operators": "off",
-    "no-tabs": "off",
-    "react/forbid-prop-types": [0],
-    "react/prefer-es6-class": 0,
-    "react/jsx-filename-extension": [1, { "extensions": [".js"] }],
-    "react/jsx-indent": ["error", "tab"],
-    "react/jsx-indent-props": ["error", "tab"],
-    "react/no-unused-prop-types": [2, { "skipShapeProps": true }]
-  },
-  "env": {
-    "es6": true,
-    "browser": true,
-    "jasmine": true,
-    "jest": true
-  },
-  "globals": {},
-  "parserOptions": {
-    "ecmaVersion": 6,
-    "sourceType": "module"
-  }
+  "extends": [
+    "@talend/eslint-config",
+    "prettier"
```

Comment:
Disable prettier rules, so it doesn't conflict with CI that run prettier

Reply:
Why not defined the prettier extends override inside the new package ?
This will remove the need for each package to have a direct dependency on a eslint plugin.
Plus the apps to have the plugin on devDependencies and also to have this extra line copied all over the .eslintrc apps.
If the reason is the need for an extra step, having a base one + the overriden one (like this file) inside the new package and export the overriden one by default, I think we should.